action/configure_networks.go

```diff
@@ -16,14 +16,15 @@ func NewConfigureNetworks(vmFinder bslcvm.Finder) ConfigureNetworks {
 	}
 }
 
-func (a ConfigureNetworks) Run(vmCID VMCID, networks bslcvm.Networks) (interface{}, error) {
+func (a ConfigureNetworks) Run(vmCID VMCID, networks Networks) (interface{}, error) {
```

Comment:
Can you motivate why your removed the `bslcmv` prefix for that import? I guess it's OK but I tend to like named imports better...